lib/MusicBrainz/Server/WebService/JSONSerializer.pm

```diff
@@ -54,6 +54,8 @@ sub work_list          { shift->entity_list(@_, "work", "works") };
 sub area_list          { shift->entity_list(@_, "area", "areas") };
 sub place_list         { shift->entity_list(@_, "place", "places") };
 sub event_list         { shift->entity_list(@_, "event", "events") };
+sub instrument_list    { shift->entity_list(@_, "instrument", "instruments") };
+sub series_list        { shift->entity_list(@_, "series", "series") };
```

Comment:
Went for the first approach. It still relies on assuming that foo-bar-list means that the element entity type name (as used in entities.json) is foo_bar though; if that can be avoided (e.g. by having that in entities.json, perhaps as a list_type property), so much the better.